addons/pos_self_order/controllers/ordering.py

```diff
@@ -0,0 +1,463 @@
+# -*- coding: utf-8 -*-
+
+from typing import List, Dict, Optional, Union
+import werkzeug
+import uuid
+
+from odoo import http, fields
+from odoo.http import request
+from odoo.tools import partition
+
+
+from odoo.addons.pos_self_order.controllers.utils import (
+    get_pos_config_sudo,
+    get_table_sudo,
+)
+from odoo.addons.pos_self_order.models.product_product import ProductProduct
+from odoo.addons.pos_self_order.models.pos_order import PosOrder, PosOrderLine
+from odoo.addons.pos_self_order.models.pos_config import PosConfig
+
+
+class PosSelfOrderController(http.Controller):
+    @http.route(
+        "/pos-self-order/send-order",
+        auth="public",
+        type="json",
+        website=True,
+    )
+    def pos_self_order_send_order(
```

Comment:
How about automatically sending the order to the kitchen? Does the waiter (pos app user) have to do it manually?

Reply:
I talked with DAMA and he said it's okay. Orders will be validated by the waiter.